### DISCUSSION TITLE
Upload zip or taz files in github repo using github api

### DISCUSSION THREAD

**Discussion by sasikumarevature** [community/community#46590] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Hi guys, 

- is there any API to up upload a zip or tar file in GitHub? if there kindly share the URL.

- second one like any API to commit multiple files using a single request ?.if there kindly share the URL.

**Comment by oscarspalk** [comment]
Using this endpoint: [create-a-blob](https://docs.github.com/en/rest/git/blobs?apiVersion=2022-11-28#create-a-blob)
You are able to upload zipfiles below the size of 100 MB.
It is not possible to upload multiple files within a single request.

**Reply by xts-bit** [reply]
@oscarspalk Can you pls help? Any code example?
